TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/206_Motion_Vectors/CaptureMotionVectorsPass.cs

```diff
@@ -0,0 +1,51 @@
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+internal class CaptureMotionVectorsPass : ScriptableRenderPass
+{
+    ProfilingSampler m_ProfilingSampler = new ProfilingSampler("MotionVecDebug");
+    Material m_Material;
+    RenderTargetIdentifier m_CameraColorTarget;
+    float m_intensity;
+
+    public CaptureMotionVectorsPass(Shader shader)
+    {
+        if (shader != null)
+            m_Material = new Material(shader);
```

Comment:
Do we destroy the material again? I dont think it breaks anything now, but hopefully we can detect leaks like this in our CI in the future